virttest/utils_v2v.py

```diff
@@ -6,15 +6,16 @@
 
 from __future__ import print_function
 
-import os
-import re
-import time
+import aexpect
```

Comment:
Why is it so above?

Reply:
Just make the module in alphanumeric order.

Reply:
Oh, ok. But `Imports should be grouped in the following order: Standard library imports. Related third party imports.` 
So maybe we  could put it a little bit lower. But nice arrangement. I could tell that you have also read the slides of Clean Code.

Reply:
oh, I forgot that. Let me update it. 
Thanks.